src/Logary.Facade/Facade.fs

```diff
@@ -405,7 +403,7 @@ module internal Formatting =
           // but using 'LevelFatal' so it really stands out.
           themedParts.Add (prop.ToString(), LevelFatal)
 
-      FsMtParser.parseParts context.Buffer template foundText foundProp
```

Comment:
It's not at all a bad idea to have a context buffer. What is the reasoning here?